src/cdk-experimental/popover-edit/edit-event-dispatcher.ts

```diff
@@ -118,7 +118,7 @@ export class EditEventDispatcher {
   private readonly _hoveredContentStateDistinct = combineLatest([
       this._getFirstRowWithHoverContent(),
       this._getLastRowWithHoverContent(),
-      this.editingOrFocused,
+      this.editingOrFocused.pipe(this._startWithNullDistinct),
```

Comment:
I think you can just change the share() at the end of the editingOrFocused declaration above to shareReplay(1)

In either case, it would be ideal to make any needed changes to the declaration unless that would lead to undesired changes